content/hi/docs/reference/glossary/persistent-volume.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Persistent Volume
```

Comment:
Better to write PersistentVolume. The words “persistent” and ”volume“ don't appear in Hindi.